### ISSUE TITLE
Automatic activities should have no assignee

### ISSUE THREAD

**Issue by Betree** [opencollective/opencollective#5135]
When an expense fails to get paid in the process CRON job, we create an activity like `EXPENSE_ERROR` with the user who triggered the initial payment action set as the `UserId`.

- This is arguably incorrect, as the activity is not created by this user but by our system
- In the frontend, it gives the impression that the error comes from this person

How things look like today:
![image](https://user-images.githubusercontent.com/1556356/150747989-97d1fd55-572d-4c3f-9253-d89ead82157f.png)

Our frontend already supports activities without an individual assigned. Here's how it would look like if we stop assigning someone to these activities:
![image](https://user-images.githubusercontent.com/1556356/150748224-0cc060a4-5f29-41dc-9e78-43779baa6d27.png)

It would be great to still show the error date in there.

@kewitz will there be any breaking change if we stop recording this `UserId` for expense errors?

**Comment by kewitz** [comment]
@Betree I don't think it will. Everything seems to work normally if I delete the `UserId` of an activity.
